addons/binding/org.openhab.binding.onkyo/ESH-INF/thing/thing-types.xml

```diff
@@ -25,6 +25,11 @@
 				<description>Port of the Onkyo to control</description>
 				<default>60128</default>
 			</parameter>
+			<parameter name="udn" type="text">
+				<label>Unique Device Name</label>
+				<description>The UDN identifies the Zone Player.</description>
+				<required>true</required>
```

Comment:
by making this mandatory, you are requiring all users that manually define their things in a file to also find out their UDN. If you made this optional, you could keep everything working as it is today, but also support audio if the UDN is set.